software_bundle/README.md

```diff
@@ -0,0 +1,219 @@
+### Software bundle management
+
+#### What is a software bundle?
+A software bundle is a single file (e.g. RPM, squashfs, zip) which can be installed
+on the system to extends the functionality of the system in some way.   
+A software bundle may be installed to:
+- fix some bug/CVE on the system, by updating the programs affected by that
+  bug/CVE
+- add a new feature to the system
+- add/update some software utility (e.g. perl, ncurses)
+- update some dependency (e.g. updating some python site-package)
+
+#### Comparison with the gNOI OS package:
```

Comment:
formatting nit:  Can you create a header for "Comparisions with other gNOI RPCs" near the end of this README and then move the OS and SetPackage comparisons into it?

Reply:
done in https://github.com/openconfig/gnoi/pull/160/commits/4df2a72eaa307b8fbead852292a1f1293b34491a